exercise/submission_models.py

```diff
@@ -28,18 +29,29 @@ def get_queryset(self):
             .prefetch_related('submitters')
 
     def create_from_post(self, exercise, submitters, request):
+
+        submission_data_list = [
+                (key, value) for (key, value) in query_dict_to_list_of_tuples(request.POST)
+                if key != '__aplus__'
+        ]
+        try:
+            meta_data_dict = json.loads(request.POST.get('__aplus__','{}'))
+        except json.JSONDecodeError:
+            raise ValueError("invalid json for __aplus__")
```

Comment:
I think `in` is a better preposition here, or is it? Or `The content of the field __aplus__ is not valid json`?